WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -76,6 +77,10 @@ class NotificationDetailsViewController: UIViewController, NoResultsViewHost {
     ///
     var nextNavigationButton: UIButton!
 
+    /// Share Footer View
+    ///
+    var shareFooterView: UIView!
```

Comment:
[Suggestion] The compiler won't complain if this property is accessed without safe unwrapping:

```swift
self.shareFooterView.removeFromSuperview() 
```

The app will crash if `shareFooterView` is `nil`.

I'd suggest to change the type of this property to `UIView?` so the compiler enforces safe unwrapping.

```swift
self.shareFooterView?.removeFromSuperview() 
```